client/containers/DashboardEdges/newEdgeEditor.scss

```diff
@@ -48,4 +49,6 @@ $margin-size: 15px;
 		margin-top: $margin-size;
 		margin-bottom: $margin-size;
 	}
+
+	margin-bottom: 30px;
```

Comment:
This is the same `30px` that's present in `PubEdgeListingCard`, right? I'm tempted to say it would be cleaner to remove both this one and that one (leaving no margin between cards on either the dashboard or the Pub-side listing) and then adding
```scss
container > * { 
    margin-bottom: 30px;
}
```
for some `container` in each of `DashboardEdgesListing` and `PubEdgeListing`. I say "tempted" because now that I've written it out I'm not sure it's actually cleaner, except that I like the idea of the parent owning the child spacing.

Food for thought, basically, feel free to ignore entirely.

Reply:
I like your idea, and I actually tried to do the same thing before and it just got a bit hairy. Definitely happy to do in a follow-up PR.